### DISCUSSION TITLE
Chromebroswer - auto browsing stops after hitting a video

### DISCUSSION THREAD

**Discussion by rbreesems** [cmu-sei/GHOSTS#102] [General]
- With Chrome V75+compatible chromebrowser control, auto browsing stops if the link followed is a video. I tried lengthening the `DelayAfter` to be longer than the video but this did not help.

- Also, an FYI, the binaries that were updated for nlog5 seem to not report timeline activities. I get Polling, NPCs on the Grafana dashboard, but not activities. With the previous binaries, activities were reported.

- Also, with the new binaries the logs/app.log is not appearing.  Logs/clientupdate is there.

**Comment by sei-dupdyke** [comment]
I just posted up new binaries fixing the nlog5 logging issues. Sorry about that, no excuse, I should have tested further after updating those references yesterday. Sorry.

Looking at video issue now.

**Comment by sei-dupdyke** [comment]
I updated and tested the video issue with chromedriver 103X and it seems to work fine for both (1) links to video services like Youtube or Vimeo (embedded videos), but also to (2) mp4 movies that play directly in the browser window.

If you are using a previous version of chromedriver, look at its logging for the video, if that doesn't fix it, please try the latest binaries, or provide me more logs and I'm happy to look into it further.

**Reply by rbreesems** [reply]
Thanks. I updated Chrome as you suggested, video problem went away.
Download the new client fixed for Nlog5,  logs/app.log is back.

I am still not getting any timeline activities showing up in my Grafana ghosts-api dashboard, but will debug that later.

I noticed during autobrowsing that Chrome tried to go to  <somedomain>/next  and this failed 404 in our synthetic environment. Is this one of the random choices from a page?

**Reply by sei-dupdyke** [reply]
There is nothing like that in the code, was it an `a href` tag somewhere in the previous page?

**Reply by sei-dupdyke** [reply]
You may want to look at something like GHOSTS Pandora for a server that always has something to return :)

https://github.com/cmu-sei/GHOSTS/tree/master/src/ghosts.pandora

**Reply by rbreesems** [reply]
Thanks for the tip, we will definitely look at that.
